api4/post.go

```diff
@@ -179,13 +182,18 @@ func getPostsForChannel(c *Context, w http.ResponseWriter, r *http.Request) {
 			c.SetInvalidParam("since")
 			return
 		}
+
+		// In case of "since" route, return all items by default for now, to maintain
+		// backwards compatibility with mobile. But when the next ESR passes, just
```

Comment:
I'd say when the version of the server that ships this becomes the ESR this can be removed, not before.

Reply:
Should I just remove the ESR-part from the comment. That should automatically means, remove this "if" block whenever the min. mobile version starts supporting the new pagination, right?